Improvements to integration tests

anoopcs9 opened the pull request:
In the light of various integration test runs(on CentOS CI) over the last few days following changes are found to be valid for reliable execution of test cases.

* Wait longer for Domain - Service IP verification
* Use different suffix for _TestScaleoutCluster_ test

anoopcs9 commented:
/retest all

spuiuk reviewed (approved):
ACK-ed by: Sachin Prabhu<sprabhu@redhat.com>

anoopcs9 commented:
/retest centos-ci/sink-clustered/mini-k8s-1.23

anoopcs9 commented:
> OK, so this looks like a subset of the changes I just commented on in PR #158. Both PRs are draft and I won't repeat myself here. Please make one or the other reviewable once you're ready and @-ping me for a review.

Sure.
Now I have rebased and fixed the commit message.

phlogistonjohn reviewed (approved):
LGTM, thanks!
